spec/src/main/asciidoc/microprofile-openapi-spec.adoc

```diff
@@ -186,35 +227,225 @@ public Response findPetsByStatus(...) { ... }
     operationId: findPetsByStatus
 ----
 
+.Sample 2 - Operation with different responses
+[source,Java]
+----
+@GET
+@Path("/{username}")
+@Operation(summary = "Get user by user name",
+        responses = {
+                @APIResponse(description = "The user",
+                        content = @Content(mediaType = "application/json",
+                                schema = @Schema(implementation = User.class))),
+                @AAPIResponse(responseCode = "400", description = "User not found")})
+public Response getUserByName(
+        @Parameter(description = "The name that needs to be fetched. Use user1 for testing. ", required = true) @PathParam("username") String username)
+ {...}
+----
+
+.Output for Sample 2
+[source, yaml]
+----
+/user/{username}:
+    get:
+      summary: Get user by user name
+      operationId: getUserByName
+      parameters:
+      - name: username
+        in: path
+        description: 'The name that needs to be fetched. Use user1 for testing. '
+        required: true
+        schema:
+          type: string
+      responses:
+        default:
+          description: The user
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+        400:
+          description: User not found
+----
 
-===== APIResponse
 
-TODO: Detailed usage description of annotation (issue #10)
+===== RequestBody
 
-.Sample 1
+
+.Sample 1 - Simple RequestBody
 [source,Java]
 ----
-
+@POST
+        @Path("/user")
+        @Operation(summary = "Create user",
+                description = "This can only be done by the logged in user.")
+        public Response methodWithRequestBodyAndTwoParameters(
+                @RequestBody(description = "Created user object", required = true,
+                        content = @Content(
+                                schema = @Schema(implementation = User.class))) User user,
```

Comment:
I think `description`, `content`, and `schema` should align.

Reply:
+1